constants/skills.js

```diff
@@ -1,14 +0,0 @@
-const headers = ['Web and Frontend', 'Backend', 'Server'];
```

Comment:
I'm sorry for the miscommunication, could we put this file back?

Reply:
We put the file back.